src/client/kotlin/tech/thatgravyboat/skyblockapi/api/data/stored/EquipmentStorage.kt

```diff
@@ -16,14 +16,13 @@ internal object EquipmentStorage {
     )
 
     private val normalEquipment: MutableMap<EquipmentSlot, ItemStack>
-        get() = EQUIPMENT.get().slots.getOrPut(ProfileAPI.profileName ?: "") {
-            EquipmentSlot.entries.associateWith { ItemStack.EMPTY }.toMutableMap()
-        }
+        get() = EQUIPMENT.get().slots.getOrPut(ProfileAPI.profileName ?: "", emptyEquipment)
 
     private val riftEquipment: MutableMap<EquipmentSlot, ItemStack>
-        get() = EQUIPMENT.get().riftSlots.getOrPut(ProfileAPI.profileName ?: "") {
-            EquipmentSlot.entries.associateWith { ItemStack.EMPTY }.toMutableMap()
-        }
+        get() = EQUIPMENT.get().riftSlots.getOrPut(ProfileAPI.profileName ?: "", emptyEquipment)
+
+    private val emptyEquipment: () -> MutableMap<EquipmentSlot, ItemStack>
```

Comment:
id prefer if this was a function instead and it had a lambda reference in the getOrPuts